ConceirgeDinning.Adapter.USRestaraunt/Models/Address.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ConceirgeDinning.Adapter.USRestaraunt.Models
+{
+    public class Address
+    {
+        public string city { get; set; }
+        public string formatted { get; set; }
```

Comment:
What does formatted describe here?

Reply:
It is according to the contract provided by the supplier....they provide data in this contract only